src/chainstate/burn/db/sortdb.rs

```diff
@@ -1103,6 +1103,11 @@ impl<'a> SortitionHandleTx<'a> {
             if let PoxAnchorBlockStatus::SelectedAndKnown(ref anchor_block, ref reward_set) =
                 next_pox_info.anchor_status
             {
+                test_debug!(
```

Comment:
Should this method explicitly check if it's in the prepare phase and return burn addresses? If I understand correctly, right now it's kind of implicit that the prepare phase is always burns, because the reward set has `(reward_cycle_length - prepare_phase_length)*2` slots.

Reply:
I went ahead and made this check explicit